### DISCUSSION TITLE
After linux kernel 5.11 upgrade, desktop fails to wake from sleep

### DISCUSSION THREAD

**Discussion by thx1111** [lxqt/lxqt#1921] [Q&A]
Arch Linux
linux 5.11.1.arch1-1
liblxqt 0.16.0-1

After upgrade from 5.10 to 5.11, on an a laptop with Intel Core2 and ATI Mobility Radeon X1600, running the lxqt desktop, sleep seems to work normally, but on wake from sleep, the desktop is frozen. The mouse still works, caps lock still works, but the tray clock is frozen, and there is no response from application windows. Non-desktop processes still work normally. For instance, ssh recovers from sleep and wake. After sleep and wake, `ps wax` does show a seemingly large number of kworker processes remaining in state I, "Idle kernel thread", of the form `[kworker/u4:21-events_unbound]`.

Reverting to the lts kernel, linux-lts 5.10.18-1, desktop processes work as expected after sleep and wake.

Because the issue correlates to the kernel version, I expect that this is a problem with the kernel.  Still, I'm wondering, has anyone else seen this desktop freeze on wake with the 5.11 kenel?

**Comment by tsujan** [comment]
I moved your report to Discussions → Q&A, although I don't think it belongs to Discussions either because it isn't related to LXQt.

Please don't use the issue tracker in this way.

EDIT: The issue tracker was sometimes used for questions before the Discussions section was added because there was no other place.

**Comment by stefonarch** [comment]
No issue here with this kernel. I suspect ATI radeon drivers. 
I'd recommend having `linux-lts` installed always too as fallback. Suspend/hibernate issues can always arise with some hardware and some kernel updates.

**Comment by thx1111** [comment]
> No issue here with this kernel. I suspect ATI radeon drivers.

Thanks for your note.  Hmm - I wonder if there is a way to test that.  I should check the status of the radeon driver after wake.

> I'd recommend having linux-lts installed always too as fallback.

Always.

**Comment by thx1111** [comment]
> I moved your report to Discussions → Q&A

Thanks for that.  I hadn't known about the Discussions area.

**Comment by thx1111** [comment]
I also did an update on a Toshiba Satellite using the i915 driver, and no problem with sleep and wake.

Another Arch user with a ryzen 4500u redmibook using the Radeon RX Vega 6 integrated GPU has also seen the problem with sleep and wake.

From ssh, on my HP with the ATI Mobility Radeon X1600, a `sudo rmmod -v -f radeon;sudo modprobe radeon` restores the display and keyboard. The original desktop is lost, of course, leaving the original Xorg process a zombie. All of those Idle kernel threads also disappear. So, yes, this seems likely to be a radeon driver issue.
